auditlog/models/rule.py

```diff
@@ -343,6 +363,12 @@ def read(self, fields=None, load="_classic_read", **kwargs):
                 return result
             self = self.with_context(auditlog_disabled=True)
             rule_model = self.env["auditlog.rule"]
+            if self.env.user in users_to_exclude_ids:
+                return result
+            for d in fields:
+                if d not in fields_to_exclude_ids:
```

Comment:
```suggestion
                if d not in fields_to_exclude:
```

But one question, the field description tells "Field to exclude", as a user I expect to not have these fields logged, but we could still have a log for the other fields linked to the request, but here you avoid the log creation as soon as one of the field to exclude is present, is it really what we want?
I would instead filter out the fields to exclude in the line above:
```python
read_values = {d["id"]: d for d in result2}
```
What do you think?